### DISCUSSION TITLE
How to run next-auth on seperated frontend and backend servers with express.js

### DISCUSSION THREAD

**Discussion by casparwamik** [nextauthjs/next-auth#3445] [Help]
Hello everyone,
The issue is the following: I have programmed a MERN full stack application, the frontend is a NextJS application, the backend is an ExpressJS API. 

The problem: I would like to use next-auth for user authentication, with jwt tokens. Since I also plan to add more frontend applications later (React-Native), I wanted to strictly detach the backend from the frontend and run it on its own server, also to have single points of failure.  

How can I run next-auth in my FullStack application on separate frontend and backend servers, and integrate it into an express backend? 

I already looked at this post, tried it, however it didn't work (probably due to new version?). 
https://github.com/nextauthjs/next-auth/issues/531 

Thanks for any support

**Comment by balazsorban44** [comment]
Check out #2294

**Comment by mohammadjavadnouri** [comment]
Same problem here!

**Comment by guptaashwanee** [comment]
I am trying to implement the same setup for my college project, did you got any success in this.
